starknet_devnet/__init__.py

```diff
@@ -1,14 +1,70 @@
 """
 Contains the server implementation and its utility classes and functions.
 """
+import logging
 import sys
 from copy import copy
+from typing import Any, Callable, Dict, Iterable, List, Optional, Tuple, cast
 
+import cairo_rs_py
 import starkware.cairo.lang.vm.crypto
 from crypto_cpp_py.cpp_bindings import cpp_hash
+from starkware.cairo.common.cairo_function_runner import CairoFunctionRunner
```

Comment:
Could `cairo_rs_py`-related patches be put into a single file which is then just imported in this `__init__.py`?

Reply:
Yes good idea, and I also create the function to patch every change at once.